src/es/components/pages/Form.html

```diff
@@ -0,0 +1,205 @@
+<!-- Figma Link: Do not copy -->
+<code><a href="https://www.figma.com/file/thNWJxDbPikhVAE95eEHLI/Design-System-Pages-%7C%C2%A0Klubschule?type=design&node-id=3111%3A289674&mode=design&t=QNrJHQav4gyyeVVi-1" target="_blank">figma</a></code>
+
+<!-- 2-spaltige Bühne mit maximaler Ausprägung -->
+<o-grid namespace="grid-2columns-content-stage-" first-container-vertical first-column-with="66%" background-color="var(--mdx-sys-color-accent-4-default)" image-src="https://mits-gossau.github.io/web-components-toolbox-klubschule/src/img/content-stage/a59843713cfd9aefb7e8d172513e4bea.png" width="100%">
+    <div class="stage-content">
+        <h1>Kontaktformular</h1>
+    </div>
+    <aside>
+        <div class="aside-image">
+            <!-- Merken-Button für Desktop -->
+            <ks-m-favorite-button off-text="Merken" on-text="Gemerkt">
+                <ks-a-button namespace="button-tertiary-" color="secondary">
+                    <a-icon-mdx icon-name="Heart" size="1em" class="icon-left"></a-icon-mdx>
+                    <span>Merken</span>
+                </ks-a-button>
+            </ks-m-favorite-button>
+        </div>
+    </aside>
+</o-grid>
+
+<!-- Angebotsliste -->
+<ks-o-body-section variant="default">
+    <ks-c-with-facet mock>
+        <o-body-style content-width-var="100%" background="transparent">
+            <o-grid namespace="grid-12er-">
+                <div col-lg="12" col-md="12" col-sm="12">
+                    <ks-o-form>
+                        <m-simple-form load-form-styles namespace="simple-form-default-">
+                            <form action="/some-action" enctype="multipart/form-data" method="post">
+                                <o-grid namespace="grid-12er-">
+                                    <div col-lg="12" col-md="12" col-sm="12">
+                                        <ks-a-heading tag="h3">Ihre Anfrage</ks-a-heading>
+                                    </div>
+                                    <div col-lg="12" col-md="12" col-sm="12">
+                                        <ks-a-input>
+                                            <div>
+                                                <label>Betreff *</label>
+                                                <input type="text" required>
+                                            </div>
+                                        </ks-a-input>
+                                    </div>
+                                    <div col-lg="12" col-md="12" col-sm="12">
+                                        <ks-a-input>
```

Comment:
remove the shadow dom, so that the element works within the form tag. to do this, ether add the attribute mode="false" or in the constructor the prop mode:'false'. https://github.com/mits-gossau/web-components-toolbox-klubschule/blob/master/src/es/components/atoms/input/Input.js#L11 example: https://github.com/mits-gossau/web-components-toolbox/blob/3cf2ca021a2662e9f16147082d48bcf5f41a4913/src/es/components/atoms/email/EMail.js#L21